contrib/golang/common/dso/dso.cc

```diff
@@ -39,18 +43,18 @@ Dso::~Dso() {
 }
 
 HttpFilterDsoImpl::HttpFilterDsoImpl(const std::string dso_name) : HttpFilterDso(dso_name) {
-  loaded_ = dlsymInternal<decltype(envoy_go_filter_new_http_plugin_config_)>(
+  dlsymInternal<decltype(envoy_go_filter_new_http_plugin_config_)>(
```

Comment:
nit: use `&=` instead of `=` is the easiest way to fix it. such as below:
```
loaded_ &= dlsymInternal<decltype(envoy_go_filter_new_http_plugin_config_)>(envoy_go_filter_new_http_plugin_config_, handler_, dso_name,
      "envoyGoFilterNewHttpPluginConfig");
```

Reply:
Thanks, done.